### PR TITLE
Update to July 5.0.205 release.

### PR DESCRIPTION
**Pull request by crummel**
-Update hashes and version numbers.
- Update ASP.NET Core patch.
- Patch MSBuild to not build samples to skip a VS prebuilt.
- Update online prebuilt baseline.

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -22,20 +22,20 @@
...
       <Uri>https://github.com/mono/linker</Uri>
-      <Sha>6a15056a9d71169bf04d007301edfd558afcfa6a</Sha>
+      <Sha>e6a47c70b7878b346ff0858085eb8a6379f6f0bb</Sha>
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.7" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
-      <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>556582d964cc21b82a88d7154e915076f6f9008e</Sha>
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="5.0.8" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
```
Comment (MichaelSimons) [inline eng/Version.Details.xml:31]: These shouldn't reference the internal repos should they?

**Review by MichaelSimons — COMMENTED** [review]
Outside of the internal repo references, everything looks good.

File: eng/Version.Details.xml (hunk 1 above)
Comment (crummel) [reply eng/Version.Details.xml]: Fixed, thanks.

File: repos/sdk.proj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <SourceDirectory>sdk</SourceDirectory>
+    <SourceDirectory>-sdk</SourceDirectory>
```
Comment (omajid) [inline repos/sdk.proj:3]: Stray `-`?

Comment (crummel) [reply repos/sdk.proj]: Thanks, fixed.

**Comment by omajid** [comment]
the `nuget-client.log` from the failing bootstrap offline job shows this error message:

/bst/bootstrap_dir/final-sdk/src/nuget.client.d525b0e670f3b6cbd5c73a35f04730a9f658c852/src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj : error NU1101: Unable to find package Microsoft.Extensions.CommandLineUtils. No packages exist with this id in source(s): prebuilt, previously-source-built, reference-packages, source-built [/bst/bootstrap_dir/final-sdk/src/nuget.client.d525b0e670f3b6cbd5c73a35f04730a9f658c852/build/restorehelper.targets]

**Comment by crummel** [comment]
Looking at this now.

File: NuGet.config [hunk 3]
```diff
@@ -2,6 +2,16 @@
...
+    <add key="darc-pub-dotnet-templating-0b9d7e7-3" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-3/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-templating-0b9d7e7-2" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-2/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-templating-0b9d7e7-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-templating-0b9d7e76-1/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-runtime-cf258a1-1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-runtime-cf258a14-1/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-roslyn-analyzers-755ceda" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-roslyn-analyzers-755ceda7/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-msbuild-5e4b48a" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-msbuild-5e4b48a2/nuget/v3/index.json" />
+    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```
Comment (dseefeld) [inline NuGet.config:14]: Are these addtional sources required for an external build?  Shouldn't everything be in dotnet-public after release?

**Review by dseefeld — APPROVED** [review]
One question on nuget sources.  Otherwise, LGTM

File: NuGet.config (hunk 3 above)
Comment (crummel) [reply NuGet.config:14]: I think you're correct, removed.

**Pull request merged**